modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications. You can upload a JAR file using the following options: 
+
+* Navigate to *+Add* view of the *Developer* perspective, click the *Upload JAR file* tile through the *From Local Machine* option. Browse and select your JAR file or drag and drop a JAR file to deploy your application.
```

Comment:
Missing article. I think it would read better if it's, `Navigate to _the_ *+Add* view...`

Reply:
Missing comma. Since it's an independent clause after `or` (subject is the understood `you` and verb is `drag or drop`) you'll want a comma before the coordinating conjunction, `or`. 

So, `Browse and select your JAR file, or drag and drop a JAR file to deploy your application`

Reply:
Thanks a lot!